config/detekt.yml

```diff
@@ -68,7 +68,7 @@ complexity:
   LongParameterList:
     active: true
     constructorThreshold: 6
-    functionThreshold: 6
+    functionThreshold: 12
```

Comment:
what does this parameter drive?

Reply:
It's how many parameters a function can have before detekt causes a `LongParameterList` warning/error (See Jonathan's comment above). I actually may reset it to 6 here and instead set `ignoreDefaultParameters` to `true`.